pkgs/applications/altcoins/exodus/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, lib, makeWrapper, fetchurl, unzip, glib, systemd, nss, nspr, gtk3-x11, gnome2, atk, cairo, gdk_pixbuf, xorg, xorg_sys_opengl, utillinux, alsaLib, dbus, at-spi2-atk, cups, vivaldi-ffmpeg-codecs, libpulseaudio }:
+
+stdenv.mkDerivation rec {
+  pname = "exodus";
+  version = "19.5.24";
+
+  src = fetchurl {
+    url = "https://exodusbin.azureedge.net/releases/${pname}-linux-x64-${version}.zip";
+    sha256 = "1yx296i525qmpqh8f2vax7igffg826nr8cyq1l0if35374bdsqdw";
+  };
+
+  sourceRoot = ".";
+  unpackCmd = ''${unzip}/bin/unzip "$src" -x "Exodus*/lib*so"'';
```

Comment:
Seems odd to use a multiline string with 1 line but I guess this removes the need to escape the quotations

Reply:
Yes, it makes it easier.

Reply:
I moved it to multiline to move it a little bit more clear.